src/components/SectionScheduleCalendar/SectionScheduleCalendar.tsx

```diff
@@ -133,6 +143,13 @@ export function SectionScheduleCalendar({
                     <CalendarEventLinkedSection interviewSection={eventDetails?.interviewSection} />
                 </ModalHeader>
                 <ModalContent className={s.SectionScheduleCalendar}>
+                    <div className={s.TimePickerWrapper}>
+                        <TimePicker
+                            value={eventDetails?.originalDate as Date}
```

Comment:
Looks dangerous cuz you casts type for probably nullish value

Reply:
I think it's safe because the component is only rendered when the slot is opened

Reply:
Just wrap with nullable, if date is always defined, it wont break anything